src/net/discovery/mock.rs

```diff
@@ -0,0 +1,16 @@
+use crate::net::client::MpcHttpConnection;
+use crate::net::discovery::PeerDiscovery;
+
+/// returns valid [`MpcHttpConnection`]s, but not pointing to real servers; only use this if the
```

Comment:
what is the use case? you don't seem to be able to do anything with this connection

Reply:
there are many tests that deal with just receiving side and not sending side. this would allow the `MpcHelperServer` struct to be created, which would require a peer list, even if the test doesn't intend to use it.